src/vendor/twitter-text-3.1.0.js

```diff
@@ -10,30 +10,33 @@
  *    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
not sure that this file should have been reformatted with our ESLint rules. Should I exclude `vendor` prefix from ESLint checker?